src/main/java/twilightforest/entity/projectile/EntityTFMoonwormShot.java

```diff
@@ -73,7 +81,9 @@ public void handleStatusUpdate(byte id) {
 		if (id == 3) {
 			int stateId = Block.getStateId(TFBlocks.moonworm.get().getDefaultState());
 			for (int i = 0; i < 8; ++i) {
-				this.world.addParticle(ParticleTypes.BLOCK_CRACK, this.getX(), this.getY(), this.getZ(), 0.0D, 0.0D, 0.0D, stateId);
+				// TODO: Don't think this is enough
+				// state ID needs to be retained somehow
+				this.world.addParticle(ParticleTypes.DAMAGE_INDICATOR, false, this.getX(), this.getY(), this.getZ(), 0.0D, 0.0D, 0.0D);
```

Comment:
this is the wrong particle type, this is the hearts that appear when you attack something.
should be something like `new BlockParticleData(ParticleTypes.BLOCK, state)`

there was also another place in this PR where you used damage_indicator that needs to be fixed too.

Reply:
Thanks, I knew this wasn't right.